### ISSUE TITLE
Legend keyboard navigation fails with useHTML

### ISSUE THREAD

**Issue by oysteinmoseng** [highcharts/highcharts#8561]
#### Expected behaviour
Using the Accessibility module, we should be able to tab to legend and press Enter/Spacebar to interact with legend items.

#### Actual behaviour
Can tab to legend, but pressing space/enter does nothing. Disabling `useHTML` for the legend fixes the issue.

Internal note: The click event is not properly faked for the element. Probably fired on the wrong element.

#### Live demo with steps to reproduce
http://jsfiddle.net/94ucpsk2/
